pkg/agent/agent.go

```diff
@@ -733,9 +742,11 @@ func (i *Initializer) initNodeLocalConfig() error {
 	}
 	transportIPv4Addr = nodeIPv4Addr
 	transportIPv6Addr = nodeIPv6Addr
+	transportName = localIntf.Name
```

Comment:
Could we name it `transportInterfaceName` to avoid confusion?

Reply:
updated